src/pages/workspaces/workspace/JobHistory.js

```diff
@@ -134,8 +147,8 @@ const JobHistory = _.flow(
   render() {
     const { namespace, name, workspace: { workspace: { bucketName } } } = this.props
     const { submissions, loading, aborting, textFilter } = this.state
-
     const filteredSubmissions = _.filter(({ asText }) => _.every(term => asText.includes(term.toLowerCase()), textFilter.split(/\s+/)), submissions)
+    const hasJobs = !_.isEmpty(filteredSubmissions)
```

Comment:
This will show the message if you search for a nonexistent string. We should check the pre-filtered list instead.